lib/python/picongpu/pypicongpu/species/operation/densityprofile/gaussian.py

```diff
@@ -0,0 +1,53 @@
+"""
+This file is part of the PIConGPU.
+Copyright 2024 PIConGPU contributors
+Authors: Brian Edward Marre, Afshari Masoud
+License: GPLv3+
+"""
+
+from .densityprofile import DensityProfile
+from .... import util
+from typeguard import typechecked
+
+
+@typechecked
+class Gaussian(DensityProfile):
+    """
+    Directional guassian density profile
+    """
+
+    gasCenterLeft = util.build_typesafe_property(float)
+    """  """
+    gasCenterRight = util.build_typesafe_property(float)
+
+    gasSigmaLeft = util.build_typesafe_property(float)
+    """the distance from gasCenterLeft until the gas density decreases to its 1/e-th part in meter"""
+
+    gasSigmaRight = util.build_typesafe_property(float)
+    """the distance from gasCenterRight until the gas density decreases to its 1/e-th part in meter"""
```

Comment:
I would suggest that we change to the following
```suggestion
    gasCenterFront = util.build_typesafe_property(float)
    """position of the front edge of the constant of middle the density profile in meter"""
    gasCenterRear = util.build_typesafe_property(float)
    """position of the rear edge of the constant of middle the density profile in meter"""

    gasSigmaFront = util.build_typesafe_property(float)
    """distance in away from gasCenterFront until the gas density decreases to its 1/e-th part in meter"""

    gasSigmaRear = util.build_typesafe_property(float)
    """The distance from gasCenterRear until the gas density decreases to its 1/e-th part in meter"""
```